sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -169,39 +198,52 @@ export namespace ConnectionContext {
       connectionContext.connection.removeAllSessions();
 
       // Close the cbs session to ensure all the event handlers are released.
-      await connectionContext.cbsSession.close();
+      await connectionContext.cbsSession.close().catch((err) => {
+        logger.verbose(
```

Comment:
The close() method already logs any error and then re-throws it. I believe we can skip logging here. Same goes for the `managementSession.close()`.  To keep things consistent, we can update the close() methods on the sender and receiver to do the necessary logging as well.

Reply:
Made all the catches no-ops, and also added logging to the sender/receiver close methods.